actions/update-deps/Dockerfile

```diff
@@ -15,7 +15,10 @@
 # TODO: figure out how to template this.
 FROM golang:1.17
 
-RUN go get github.com/google/ko@v0.6.0 github.com/google/licenseclassifier github.com/google/go-licenses github.com/dprotaso/modlog knative.dev/test-infra/buoy
+RUN go install github.com/google/ko@latest
```

Comment:
Should this have been `@0.6.0` or do we not care?

Reply:
For update dependencies, ko shouldn't be relevant.

Do we really need ko for update deps?